playbooks/README.md

```diff
@@ -18,6 +19,7 @@ and are nominally put in the order of usage with some exceptions. This may be us
 | `create_cluster.yml`                   | Creates cluster definition in assisted installer              | -                     |
 | `generate_discovery_iso.yml`           | Generate descovery iso                                        | `cluster_id`          |
 | `boot_iso.yml`                         | Reboots nodes to discovery iso                                | -                     |
+| `mount_discovery_iso_for_pxe.yml`      | crate the files for the PXE                                   | -                     |
```

Comment:
Consider changing to something like: 
"Creates files for PXE, mounts discovery iso and configures grub"

Reply:
I've just noticed something can you move this line above `boot_iso` to preserve the order of execution

Reply:
I agree that is not a good name but this play does:
1. Mount the iso.
2. Upload the files to the http-store.
3. Make the grub file.
4. Download the files to the TFTP server
Maybe we will split that to more plays

Reply:
It's true that the name of the playbook and role is less than perfect, but I don't have a good suggestion on how to fix it.

I think the least what we could do is to provide as much clarification in the description as possible. Based on the flow you've described, what about changing this entry in the table to: 
```
`mount_discovery_iso_for_pxe.yml` | Mounts discovery iso, generates grub config and syncs files to the TFTP server
```

Reply:
Please let me know if you think the proposed description is an improvement. This request for changes has the lowest priority and I'm happy to drop it if you disagree with the direction.

Reply:
DONE